src/api/payment-requests/payment-requests.md

```diff
@@ -301,13 +302,14 @@ Payment Activities are created when a Payment Request has been **created**, **pa
   }
 {% endreqspec %}
 
-{% h4 Fields %}
+{% h4 Post Request Fields %}
 
 |        Field         |       Type        |                                                                               Description                                                                                |
 | :------------------- | :---------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | configId             | String            | The [Merchant Config][] id used to configure the payment options.                                                                                                        |
 | value                | {% dt Monetary %} | The canonical value of the payment request. Must be positive.                                                                                                            |
 | barcode              | String {% opt %}  | The patron's barcode to identify the account to attach the payment request to                                                                                            |
+| barcodeHint          | String {% opt %}  | **EXPERIMENTAL** Type of barcode provided. Required when barcode is present. Values are "farmlands" or "centrapay".                                                      |
```

Comment:
Making this required when barcode is present will break the API contract.
And does paypal pass through a barcode when making a payment request, how will this effect them.